docs/Maintainers.md

```diff
@@ -1,20 +1,21 @@
-Community Maintenance Committee
-===============================
+Maintainers Group
+=================
 
-This document describes the Community Maintenance Committee of Open Enclave. By
+This document describes the Open Enclave Maintainers Group, which in some open source
+projects is also known as a Technical Steering Committee. By
```

Comment:
This is probably worth discussing in a meeting, since the rest of this doc doesn't talk about the roles and responsibilities as it pertains to a Technical Steering Committee.

Reply:
Yeah this is the only change I'm not entirely sure about.

Reply:
Removed mention